Apps/RTSPtoWeb/RTSPtoWeb.yaml

```diff
@@ -0,0 +1,32 @@
+name: rtsp-to-web
+services:
+    rtsp-to-web:
+        cpu_shares: 90
+        command: []
+        container_name: RTSPtoWeb
+        deploy:
+            resources:
+        image: cmer/rtsp-to-web
+        labels:
+            icon: https://static.vecteezy.com/system/resources/previews/009/882/145/original/video-surveillance-icon-cctv-camera-free-vector.jpg
+        network_mode: bridge
+        ports:
+            - target: 8083
+              published: "8083"
+              protocol: tcp
+        restart: unless-stopped
+        volumes:
+            - type: bind
+              source: /DATA/AppData/rtsp-to-web
+              target: /tmp
+x-casaos:
```

Comment:
Can you look at: https://github.com/bigbeartechworld/big-bear-casaos/blob/8e00ebbee8eb936f1a44f888beea8a26570d06d1/Apps/homarr/docker-compose.yml#L49